### PR TITLE
localize the status page some more

### PR DESCRIPTION
**Pull request by christf**
The statuspage contains some contents that are not localized and german translations are missing.
This should address the issue.

### CODE REVIEW

File: package/gluon-status-page/src/js/lib/helper.js [hunk 1]
```diff
@@ -106,7 +106,9 @@ define([ "bacon" ], function (Bacon) {
         "not connected": "nicht verbunden",
         "Packets/s": "Pakete/s",
         "Statistic": "Statistik",
-        "Neighbors": "Nachbarknoten"
+        "Neighbors": "Nachbarknoten",
```
Comment (mweinelt) [inline package/gluon-status-page/src/js/lib/helper.js]: korrekt wäre Nachbarn, heißt ja auch nicht Neighbouring Nodes im englischen.

Comment (christf) [reply package/gluon-status-page/src/js/lib/helper.js]: danke fürs review.

Comment (mweinelt) [reply package/gluon-status-page/src/js/lib/helper.js]: Ansonsten könnte man das auch Neighbourhood/Nachbarschaft nennen, fände ich nicht verkehrt.

Comment (christf) [reply package/gluon-status-page/src/js/lib/helper.js]: Nachbarn find ich von den Varianten am Griffigsten und am Besten. Update liegt vor.

**Comment by mweinelt** [comment]
Die Commit Message sollte das Modul nennen, also bspw.:
`gluon-status-page: improve localization`

**Comment by christf** [comment]
cool - ich kann in dem commit herumschreiben und der ursprüngliche Autor geht trotzdem nicht verloren. :)

File: package/gluon-status-page/src/js/lib/helper.js [hunk 2]
```diff
@@ -106,7 +106,9 @@ define([ "bacon" ], function (Bacon) {
         "not connected": "nicht verbunden",
         "Packets/s": "Pakete/s",
         "Statistic": "Statistik",
-        "Neighbors": "Nachbarknoten"
```
Comment (MTRNord) [inline package/gluon-status-page/src/js/lib/helper.js]: Why changing this to "Nachbarn"?

Comment (NeoRaider) [reply package/gluon-status-page/src/js/lib/helper.js]: I don't really see the point either, good translations are always about context.  "Nachbarknoten" sounds good, it doesn't need to match English version exactly.

File: package/gluon-status-page/src/js/lib/gui/nodeinfo.js [hunk 3]
```diff
@@ -39,8 +50,8 @@ define(["lib/helper"], function (Helper) {
       dlEntry(list, nodeInfo, "network.mac", Helper._("Primary MAC"))
       dlEntry(list, nodeInfo, "network.addresses", Helper._("IP Address"))
       dlEntry(list, nodeInfo, "software.firmware.release", Helper._("Firmware"))
-      dlEntry(list, nodeInfo, "software.fastd.enabled", "Mesh-VPN")
-      dlEntry(list, nodeInfo, "software.autoupdater.enabled", Helper._("Automatic updates"))
+      dlEntry(list, nodeInfo, "software.fastd.enabled", "Mesh-VPN", enabledDisabled)
```
Comment (NeoRaider) [inline package/gluon-status-page/src/js/lib/gui/nodeinfo.js]: If we're touching this, Mesh-VPN should get localized as well. The English version should be "Mesh VPN".

**Comment by christf** [comment]
quick poll for "Nachbarknoten" vs. "Nachbarn".
reat with thumbs up if you like "Nachbarn"
React with thumbs down if you like "Nachbarknoten".

I will not change this word more than one more time. :)

File: package/gluon-status-page/src/js/lib/helper.js (hunk 2 above)
Comment (christf) [reply package/gluon-status-page/src/js/lib/helper.js]: :)

**Comment by christf** [comment]
The poll was closed and the re-work mentioned in this discussion has been done.

**Pull request merged**